cfme/tests/storage/test_volume_backup.py

```diff
@@ -125,12 +125,82 @@ def test_storage_volume_backup_restore(backup):
             1.
             2. check in Task whether restored
     """
-    pass
+    backup.restore(name=backup.volume, new_volume=False)
+    backup.refresh()
+    volumes_collection = appliance.collections.volumes
+    restored_volume = volumes_collection.instantiate(name=backup.volume, provider=provider)
+    wait_for(lambda: restored_volume.status == 'available', fail_func=backup.refresh, delay=30,
+             timeout=600)
```

Comment:
In order for this test to assert properly I think this should be like,

```python
try:
    wait_for(lambda: restored_volume.status == 'available', fail_func=backup.refresh, delay=30,
             timeout=600)
except TimedOutError:
    pytest.fail("Failed waiting for volume to become available")
```

But I'd consider it _optional_ for now.